rollbar/test/test_rollbar.py

```diff
@@ -804,9 +805,29 @@ def test_modify_arg(self, send_payload):
 
         frames = payload['data']['body']['trace']['frames']
         called_with_frame = frames[1]
-        
+
         self.assertEqual('changed', called_with_frame['args'][0])
 
+    @mock.patch('rollbar.send_payload')
+    def test_unicode_exc_info(self, send_payload):
+        unicode_char = '\u221a'
+
+        if sys.version_info[0] < 3:
+            message = unicode_char.decode('unicode-escape')
+            expected = '\xe2\x88\x9a'
+        else:
+            message = unicode_char
+            expected = message
```

Comment:
I think we should change `_to_str()` to make sure it is always returning a `str` that is encoded in UTF-8.